rename 'runtime' interface name to 'store' (#14)
WebAssembly/wasi-runtime-config: imports.md

<h1><a id="imports"></a>World imports</h1>
<ul>
<li>Imports:
<ul>
<li>interface <a href="#wasi_config_store_0_2_0_draft"><code>wasi:config/store@0.2.0-draft</code></a></li>
</ul>
</li>
</ul>
<h2><a id="wasi_config_store_0_2_0_draft"></a>Import interface wasi:config/store@0.2.0-draft</h2>
<hr />
<h3>Types</h3>
<h4><a id="error"></a><code>variant error</code></h4>

> <p>An error type that encapsulates the different errors that can occur fetching config</p>

Reviewer: ```suggestion
<p>An error type that encapsulates the different errors that can occur fetching configuration values.</p>
```

</ul>
<hr />
<h3>Functions</h3>
<h4><a id="get"></a><code>get: func</code></h4>

> <p>Gets a single opaque config value set at the given key if it exists</p>

Reviewer: ```suggestion
<p>Gets a string configuration value if the key exists. If the key does not exist, the value returned will be None.</p>
```

<h5>Params</h5>
<ul>
<li><a id="get.key"></a><code>key</code>: <code>string</code></li>
</ul>
<h5>Return values</h5>
<ul>
<li><a id="get.0"></a> result&lt;option&lt;<code>string</code>&gt;, <a href="#error"><a href="#error"><code>error</code></a></a>&gt;</li>
</ul>
<h4><a id="get_all"></a><code>get-all: func</code></h4>

> <p>Gets a list of all set config data</p>

Reviewer: ```suggestion
<p>Gets a list of each configuration key and value.</p>
```
